pyproject.toml

```diff
@@ -6,16 +6,19 @@ authors = ["IVPK <atviriduomenys@ivpk.lt>"]
 
 [tool.poetry.dependencies]
 python = "^3.10"
-Django = "^3.2.13"
-django-cms = "^3.10.0"
+Django = "^3.2.14"
+django-cms = "^3.10.1"
 django-sekizai = "^3.0.1"
 django-filer = "^2.2.1"
 djangocms-text-ckeditor = "^5.0.1"
 djangocms-blog = "^1.2.3"
 django-compressor = "^4.0"
 django-libsass = "^0.9"
 libsass = "^0.21.0"
-django-sass-processor = "^1.2"
+django-sass-processor = "^1.2.1"
+mysqlclient = "^2.1.1"
+django-environ = "^0.9.0"
+psycopg2 = "^2.9.3"
```

Comment:
I would recommend precompiled binaries, that comes with [psycopg2-binary](https://pypi.org/project/psycopg2-binary/) package.